tests/components/rfxtrx/test_sensor.py

```diff
@@ -35,11 +34,11 @@ async def test_one_sensor(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    assert 1 == len(rfxtrx_core.RFX_DEVICES)
-    entity = rfxtrx_core.RFX_DEVICES["sensor_05_02"]["Temperature"]
-    assert "Test Temperature" == entity.name
-    assert TEMP_CELSIUS == entity.unit_of_measurement
-    assert entity.state is None
+    state = hass.states.get("sensor.test_temperature")
+    assert state
+    assert state.state == "unknown"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)